Dockerfile

```diff
@@ -1,20 +1,13 @@
 FROM ubuntu:16.04
 
-LABEL maintainer="zyxep"
+RUN apt-get update
+RUN apt-get install -y wget curl
+RUN /bin/bash -c "$(wget -qO- https://apt.vapor.sh)"
```

Comment:
I added wget as this is the recommended way on the Vapor docs. https://docs.vapor.codes/2.0/getting-started/install-on-ubuntu/#dockerfile

Reply:
Logan and tanner should have a slap! They use both wget and curl in that, totally not needed

Reply:
Using curl instead